checker/build.xml

```diff
@@ -510,6 +520,15 @@
     <!-- Used to guarantee that targets executed from other ant scripts do not cause unnecessary rebuilds -->
     <target name="all-tests-nojar" depends="build-tests"
             description="Run tests for all checkers, WITHOUT building anything">
+
```

Comment:
I agree with this.  I just haven't had time since I am trying to focus on the Verigames experiment we're trying to run.  I'll see if this can be done quickly but if not, I will add an issue and move on.